src/sidebar/components/Chat/Message/index.tsx

```
@@ -87,179 +93,199 @@ const MessageCard = memo(
-                    <Button
-                      isIconOnly
-                      radius="full"
-                      size="sm"
-                      variant="flat"
-                      isDisabled={!canRegenerate}
-                      onPress={onRegenerate}
+                >
+                  <ContextField messageContext={messageContext} />
+                  <ToolInvocations tools={tools} />
+                  <div ref={messageRef} className="text-small">
+                    {typeof message === 'string' ? <Markdown children={message} /> : message}
```

Biome

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)